.github/workflows/auto-approve.yml

```diff
@@ -0,0 +1,11 @@
+name: Auto approve
+on: pull_request
+
+jobs:
+  build:
```

Comment:
This name is weird for this job. Perhaps "approve" would be better?

Also, consider giving the job a `name:` for the logs that's clearer, like "Auto-approve dependabot diff"